src/itype.cpp

```diff
@@ -9,6 +9,12 @@
 #include <algorithm>
 #include <cmath>
 
+std::string gunmod_location::name() const
+{
+    // Yes, currently the name is just the translated id.
+    return _( _id.c_str() );
```

Comment:
Why not? It needs to be translated, just like it is translated now, see changes in item.cpp.